### PR TITLE
Add generic OSG site draining instructions (SOFTWARE-4126)

### PR DESCRIPTION
**Pull request by brianhlin**
@djw8605 I could use a once over on the Slurm instructions

### CODE REVIEW

File: docs/compute-element/hosted-ce.md [hunk 1]
```diff
@@ -38,7 +38,8 @@ Before preparing your cluster for OSG resource requests, consider the following
     see [this section](#htcondor-clusters-only-installing-the-osg-worker-node-client) for more details.
 -   [Temporary scratch space](../worker-node/using-wn.md#for-site-administrators) on each worker node;
     site administrators should ensure that files in this directory are regularly cleaned out.
--   OSG resource contributors must inform the OSG of [any relevant](../site-responsibilities.md) changes to their site.
+-   OSG resource contributors must inform the OSG of [any relevant](../site-maintenance.md##notify-osg-of-major-changes)
```
Comment (matyasselmeci) [inline docs/compute-element/hosted-ce.md:41]:
```suggestion
-   OSG resource contributors must inform the OSG of [any relevant](../site-maintenance.md#notify-osg-of-major-changes)
```

File: docs/site-maintenance.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+1.  [Let us know](#help) so that we can track down the offending software or user:
+    the same issue that you're experiencing may also be affecting other sites!
+
+Keep OSG Software Updated
+-------------------------
+
+It is important to keep your software and data (e.g., CAs and VO client) up-to-date with the latest OSG release.
+See the release notes for your installed release series:
+
+-  [OSG 3.5 release notes](release/notes.md)
```
Comment (matyasselmeci) [inline docs/site-maintenance.md:54]: Should we make a `release/osg-35.md -> notes.md` symlink and use that instead?

**Review by matyasselmeci — APPROVED** [review]
Other than the one typo, looks good to me.

File: docs/site-maintenance.md (hunk 2 above)
Comment (brianhlin) [reply docs/site-maintenance.md:54]: We could add a redirect instead of a symlink

Comment (matyasselmeci) [reply docs/site-maintenance.md:54]: Sure

**Review by djw8605 — APPROVED** [review]
Slurm stuff looks good to me.  For the partition down, it's a simple enough instruction, I wonder if you should just include it in the doc.

**Pull request merged**